src/core_plugins/kibana/server/tutorials/kubernetes_logs/index.js

```diff
@@ -0,0 +1,185 @@
+  },
+  DEPLOY: {
+    title: 'Deploy to Kuberentes',
+    textPre: 'Deploy the configured manifest to Kuberentes:',
+    commands: [
+      'kubectl create -f filebeat-kubernetes.yaml'
+    ]
+  },
+};
+
+export function kubernetesLogsSpecProvider() {
+  const moduleName = 'kubernetes';
+  const platforms = ['OSX', 'DEB', 'RPM'];
+  return {
+    id: 'kubernetesLogs',
+    name: 'Kubernetes logs',
+    category: TUTORIAL_CATEGORY.LOGGING,
+    shortDescription: 'Fetch logs from your containers running on Kubernetes.',
+    longDescription: 'The `kubernetes` Filebeat manifest fetches logs from Kubernetes containers.' +
+                     ' [Learn more]({config.docs.beats.filebeat}/running-on-kubernetes.html).',
+    euiIconType: 'logoKubernetes',
+    artifacts: {
+      application: {
+        label: 'Discover',
+        path: '/app/kibana#/discover'
+      },
+      dashboards: [],
+      exportedFields: {
+        documentationUrl: '{config.docs.beats.filebeat}/exported-fields-kubernetes-processor.html'
+      }
+    },
+    completionTimeMinutes: 10,
+    onPrem: onPremInstructions(moduleName, platforms),
+    elasticCloud: cloudInstructions(moduleName, platforms),
+    onPremElasticCloud: onPremCloudInstructions(moduleName, platforms)
+  };
+}
+
+
+
+function onPremInstructions(moduleName, platforms) {
+  const variants = [];
+  for (let i = 0; i < platforms.length; i++) {
+    const platform = platforms[i];
+    const instructions = [];
+    instructions.push(KUBERNETES_INSTRUCTIONS.INSTALL);
+    instructions.push(KUBERNETES_INSTRUCTIONS.CONFIG);
+    instructions.push(KUBERNETES_INSTRUCTIONS.DEPLOY);
+    variants.push({
+      id: INSTRUCTION_VARIANT[platform],
+      instructions: instructions
+    });
+  }
+  return {
+    instructionSets: [
+      {
+        title: 'Getting Started',
+        instructionVariants: variants,
+        statusCheck: filebeatStatusCheck()
+      }
+    ]
+  };
+}
+
+function onPremCloudInstructions(moduleName, platforms) {
+  const variants = [];
+  for (let i = 0; i < platforms.length; i++) {
+    const platform = platforms[i];
+    variants.push({
+      id: INSTRUCTION_VARIANT[platform],
+      instructions: [
+        TRYCLOUD_OPTION1,
+        TRYCLOUD_OPTION2,
+        KUBERNETES_INSTRUCTIONS.INSTALL,
+        KUBERNETES_INSTRUCTIONS.CONFIG,
+        KUBERNETES_INSTRUCTIONS.DEPLOY
+      ]
+    });
+  }
+
+  return {
+    instructionSets: [
+      {
+        title: 'Getting Started',
+        instructionVariants: variants,
+        statusCheck: filebeatStatusCheck()
+      }
+    ]
+  };
+}
+
+function cloudInstructions(moduleName, platforms) {
+  const variants = [];
+  for (let i = 0; i < platforms.length; i++) {
+    const platform = platforms[i];
+    variants.push({
+      id: INSTRUCTION_VARIANT[platform],
+      instructions: [
+        KUBERNETES_INSTRUCTIONS.INSTALL,
+        KUBERNETES_INSTRUCTIONS.CONFIG,
```

Comment:
It would be better to use the `cloud.id` for the cloud instructions. That's especially convenient here because Kibana can fill in the `cloud.id` for you.

To test the cloud instructions, put this in your kibana config:

```
xpack.cloud.enabled: true
xpack.cloud.id: hello-test
```